vector/src/main/java/im/vector/activity/VectorRoomCreationActivity.java

```diff
@@ -365,6 +361,87 @@ public static String isDirectChatRoomAlreadyExist(String aUserId, MXSession mSes
         return null;
     }
 
+
+    //==============================================================================================================
+    // Handle open existing direct chat room
+    //==============================================================================================================
+
+    /**
+     * Open the current direct chat with the corresponding user id.
+     *
+     * @param item : participant id
+     * @param canCreate create the direct chat if it does not exist.
+     * @return the corresponding room id or null
+     */
+    private boolean openDirectChat (final ParticipantAdapterItem item, boolean canCreate) {
+        Room existingRoom = this.isDirectChatRoomAlreadyExist(item.mUserId, mSession, false);
+        boolean directChatOpened = false;
```

Comment:
this boolean should be renamed: `isDirectChatOpened`, or `isDirectChatFound`. I would prefer `succeeded`